src/bapsf_motion/actors/axis_.py

```diff
@@ -136,13 +136,13 @@ def stop_running(self, delay_loop_stop=False):
     @property
     def config(self) -> Dict[str, Any]:
         """Dictionary of the axis configuration parameters."""
-        _config = {
+        return {
             "name": self.name,
             "ip": self.motor.ip,
             "units": str(self.units),
             "units_per_rev": self.units_per_rev.value.item()
         }
-        return _config
+    config.__doc__ = BaseActor.config.__doc__
```

Comment:
Since Sphinx's `autoattribute` and `autoproperty` does not retrieve the parent docstring for an attribute/property that is overwritten and decorated, this is a crude workaround to make the base class docstring show up.

Note: This is not true for python itself.  If the overwritten attribute/property does not have an overwriting docstring, then python will inspect for the parent's docstring.